Playbooks/AS-IP-Blocklist/azuredeploy.json

```diff
@@ -0,0 +1,373 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "AS-IP-Blocklist", 
+        "description": "This playbook is intended to be run from a Microsoft Sentinel Incident. It will add the IP address from Microsoft Sentinel Incidents to a Microssoft Azure Conditional Access Named Locations list, signifying compromised IP addresses.",
+        "prerequisites": "1. A Microsoft Azure Named Locations list id is needed. 2. An Microsoft Azure App Registration is also needed for the Microwsoft Graph API. 3. A Microsoft Azure key vault containing the app registration secret must also be set up. Support for the set up and configuration of each of these items can be found here: https://github.com/Accelerynt-Security/AS-IP-Blocklist",
```

Comment:
1.  "An Microsoft Azure App", I think it should be "A Microsoft Azure AD App"
2.  Please correct spelling "Microwsoft"

Reply:
Updates have been made. Thank you.